tests/engine/results/test_results.py

```diff
@@ -120,3 +129,65 @@ def test_exists_for_json_objs(self, value):
         result = PrefectResult()
         assert result.exists(json.dumps(value)) is True
         assert result.exists(value) is False
+
+
+class TestLocalResult:
+    @pytest.fixture(scope="class")
+    def tmp_dir(self):
```

Comment:
No need to change this but FYI `pytest` offers a builtin fixture for this pattern `tmpdir` (no underscore) that you can use.

Reply:
TIL